core/math/math.sgl

```diff
@@ -19,4 +19,18 @@ var pi = 3.1415926535897932;
 var sqrt1_2 = 0.7071067811865476;
 
 // Square root of 2
-var sqrt2 = 1.4142135623730951;
\ No newline at end of file
+var sqrt2 = 1.4142135623730951;
+
+function min(a, b) {
+    if (b < a) return b;
+    else return a;
+}
+
+function max(a, b) {
+    if (b > a) return b;
+    else return a;
+}
+
+function remap(value, start1, stop1, start2, stop2) {
+    return (value - start1) / (stop1 - start1) * (stop2 - start2) + start2;
```

Comment:
The order doesnt quite make sense although it maybe correct.

Id suggest `start2 + (value - start1) * (stop2 - start2) / ...`

Also, the names dont make much sense either as we are referring to a range (lower/upperbound). Ive seen `low` and `high` often used

Reply:
> The order doesnt quite make sense although it maybe correct.

It is correct. This function is also available in p5.js called as `map`. Most other languages call it remap as map is also a thing. Which is not in js.

Reply:
I know its correct, rearrange the order of the expression so that its purpose is more obvious